README.md

```diff
@@ -4,59 +4,69 @@ Elliptic curve crypto in python including secp256k1 and alt_bn128
 
 [![GitHub](https://img.shields.io/github/license/mashape/apistatus.svg)](https://github.com/ethereum/py_ecc) [![Build Status](https://circleci.com/gh/ethereum/py_ecc.svg?style=shield)](https://circleci.com/gh/ethereum/py_ecc) [![PyPI version](https://badge.fury.io/py/py-ecc.svg)](https://badge.fury.io/py/py-ecc)
 
-
 ## Quickstart
+
 ```sh
 pip install py_ecc
 ```
 
 ## BLS Signatures
 
-```python
-from py_ecc import bls
+`py_ecc` implements the [IETF BLS draft standard v0](https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-00) with [hash-to-curve v5](https://tools.ietf.org/html/draft-irtf-cfrg-hash-to-curve-05) as per the inter-blockchain standardization agreement. The BLS standards specify [different ciphersuites](https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-00#section-4.2) which each have different functionality to accommodate various use cases. The following ciphersuites are availible from this library:
+
+- `G2Basic` also known as `BLS_SIG_BLS12381G2-SHA256-SSWU-RO-_NUL_`
+- `G2MessageAugmentation` also known as `BLS_SIG_BLS12381G2-SHA256-SSWU-RO-_AUG_`
+- `G2ProofOfPossession` also known as `BLS_SIG_BLS12381G2-SHA256-SSWU-RO-_POP_`
 
-domain = 43
+### Basic Usage
+
+```python
+from py_ecc.bls.G2ProofOfPossession import (
+    PrivToPub
+    Sign,
+    Verify,
+    Aggregate,
+    FastAggregateVerify,
+    AggregateVerify,
+)
```

Comment:
It doesn't work since `G2ProofOfPossession` is a class.
We can either 
1. `from py_ecc.bls import G2ProofOfPossession as bls_pop` and then use `bls_pop.PrivToPub`, or
2. Move `G2Basic`, `G2MessageAugmentation`, `G2ProofOfPossession` to different files, and add APIs interface like:
```python
PrivToPub = G2ProofOfPossession.PrivToPub
Sign = G2ProofOfPossession.Sign
Verify = G2ProofOfPossession.Verify
Aggregate = G2ProofOfPossession.Aggregate
FastAggregateVerify = G2ProofOfPossession.FastAggregateVerify
AggregateVerify = G2ProofOfPossession.AggregateVerify
```

Reply:
Good catch! 👀 I think I'll go with option 1. Implementing now.